client/src/components/Slider/index.jsx

```diff
@@ -33,7 +33,7 @@ export default function Slider({ cards }) {
       modules={[Autoplay]}
       className={styles.swiper}
       slideClass={styles.swiperSlide}
-      loop={cards.length !== 2}
+      loop={cards.length >= 4}
```

Comment:
I made that changes because we don't want to see duplicates card when user search only for 1 item. So if there less than 4 items it stops looping